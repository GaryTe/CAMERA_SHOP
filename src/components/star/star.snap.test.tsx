import {render} from '@testing-library/react';
import Star from './star';

const number = 3;

describe('Component: Star', () => {
  it('should render correctly Star', () => {
    const {container} = render(<Star result={number} />);

    expect(container).toMatchSnapshot();
  });
});
