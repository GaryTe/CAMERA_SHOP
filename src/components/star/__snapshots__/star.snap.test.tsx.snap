// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Star should render correctly Star 1`] = `
<div>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      xlink:href="#icon-full-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
  <svg
    aria-hidden="true"
    height="16"
    width="17"
  >
    <use
      xlink:href="#icon-star"
    />
  </svg>
</div>
`;
